packages/create-cloudflare/src/frameworks/angular/templates/tools/alter-polyfills.mjs

```diff
@@ -0,0 +1,32 @@
+import { EOL } from "node:os";
+import fs from "node:fs";
+import { join } from "node:path";
+import { worker } from "./paths.mjs";
+
+/**
+ * Split by lines and comment the banner
+ * ```
+ * import { createRequire } from 'node:module';
+ * globalThis['require'] ??= createRequire(import.meta.url);
+ * ```
+ */
+const serverPolyfillsFile = join(worker, "polyfills.server.mjs");
+const serverPolyfillsData = fs
+  .readFileSync(serverPolyfillsFile, "utf8")
+  .split(EOL);
+
+for (let index = 0; index < 2; index++) {
+  if (serverPolyfillsData[index].includes("createRequire")) {
+    serverPolyfillsData[index] = "// " + serverPolyfillsData[index];
+  }
+}
+
+// Add needed polyfills
+serverPolyfillsData.unshift(
+  `globalThis['process'] = {};`,
+  `globalThis['global'] = globalThis;`,
+  // Needed as performance.mark is not a function in worker.
+  `performance.mark = () => {};`,
```

Comment:
This seems to be incorrectly polyfilled by wrangler. Which caused `performance.mark` if not a function error.

Since this is not a Node API, do you reckon it can be fixed from your end?

Reply:
//cc @petebacondarwin

Reply:
Currently `mark()` is not supported:
 https://developers.cloudflare.com/workers/runtime-apis/performance/

Looks like `performance.now()` was added earlier this year - see https://github.com/cloudflare/workerd/pull/603.

So perhaps a feature request for `performance.mark()` might be accepted?

cc @jasnell